app/helpers/import_export_helper.rb

```diff
@@ -298,13 +298,31 @@ def build_excel_export_for_patients(patients)
     end
   end
 
-  def get_comprehensive_details(patients, patient_statuses)
-    comprehensive_details = get_incomplete_comprehensive_details(patients)
-    patients_jurisdiction_paths = get_jurisdiction_paths(patients)
-    patients_labs = get_latest_labs(patients)
+  # Patient fields relevant to linelist export
+  def linelists_for_export(patients, statuses)
+    linelists = incomplete_linelists_for_export(patients)
+    patients_jurisdiction_names = jurisdiction_names(patients)
+    patients_transfers = latest_transfers(patients)
+    patients.each do |patient|
+      linelists[patient.id][:jurisdiction] = patients_jurisdiction_names[patient.id]
+      linelists[patient.id][:status] = statuses[patient.id]&.gsub('exposure ', '')&.gsub('isolation ', '')
+      next unless patients_transfers[patient.id]
+
+      linelists[patient.id][:transferred_at] = patients_transfers[patient.id][:transferred_at]
```

Comment:
actually might be able to just do:

`linelist[patient.id] = patients_transfers[patient.id]`

I was refactoring some of this code after the addition of the new linelist fields and didn't notice that this could be simplified now whereas it might not have been the case before. thanks for noticing this!

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/commit/7f06c26894cd5271163119a8b04b909943f4bd46#diff-49a8fd96370c076f87a96f077a708088R311